navig/MsgOdometry odometry

float64 direction
float64 bearing

navig/MsgLocalPosition position
navig/MsgPoint2d center

float64 left_bottom
float64 right_bottom

float64 distance
float64 width_ratio

int32 frame_number